### PR TITLE
[Place Page Chart] PM 2.5

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: server/chart_config/climate.json [hunk 1]
```diff
@@ -54,5 +54,18 @@
     ],
     "unit": "celsius",
     "isOverview": false
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Mean_concentration_PM2.5",
+    "title": "Mean concentration of PM 2.5",
```
Comment (beets) [inline server/chart_config/climate.json:61]: " Mean PM2.5 concentration" seems to be the more common way to describe this.

File: server/chart_config/climate.json [hunk 2]
```diff
@@ -54,5 +54,18 @@
...
+    "description": "Mean concentration of PM 2.5",
```
Comment (beets) [inline server/chart_config/climate.json:62]: please be more descriptive for this.. maybe add that it's for air quality, and measures concentrations of fine inhalable particles, with diameters that are generally 2.5 micrometers and smaller in the air. you could add this link too: https://www.epa.gov/air-trends/particulate-matter-pm25-trends 

this doesn't get displayed -- and goes straight to the translators

File: static/js/i18n/strings/en/units.json [hunk 3]
```diff
@@ -34,5 +34,11 @@
   },
   "celsius-display": {
     "defaultMessage": "°c"
+  },
+  "micro-gram-per-cubic-meter": {
+    "defaultMessage": "{0} μg/m3"
```
Comment (beets) [inline static/js/i18n/strings/en/units.json:39]: should we use the superscript character for this? µg/m³

File: static/js/i18n/strings/en/units.json [hunk 4]
```diff
@@ -34,5 +34,11 @@
   },
   "celsius-display": {
     "defaultMessage": "°c"
```
Comment (beets) [inline static/js/i18n/strings/en/units.json:36]: sorry just noticed this. should be °C (capital C)

**Review by beets — APPROVED** [review]
thanks - this is a good chart to add

File: server/chart_config/climate.json (hunk 1 above)
Comment (shifucun) [reply server/chart_config/climate.json]: Done

File: server/chart_config/climate.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config/climate.json]: Done

File: static/js/i18n/strings/en/units.json (hunk 3 above)
Comment (shifucun) [reply static/js/i18n/strings/en/units.json]: Done

File: static/js/i18n/strings/en/units.json (hunk 4 above)
Comment (shifucun) [reply static/js/i18n/strings/en/units.json]: Good catch!

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**